tensorflow_addons/image/gaussian_filter_ops.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+"""GaussuanBlur Op"""
+
+
+import tensorflow as tf
+from tensorflow_addons.utils.types import FloatTensorLike
+
+
+def gaussian_blur(
+    img: FloatTensorLike, sigma: tf.float64, kSize: tf.float64
```

Comment:
`FloatTensorLike` for `sigma`, and `int` for `kernel_size`.

Reply:
Done!!